docs/README.md

```diff
@@ -0,0 +1,624 @@
+
+#### Vintage
+
+Year merchant entered the market
+
+</details>
+
+### Metrics
+
+Base metric class.
+    
+
+<details markdown='1'><summary>Metrics</summary>
+
+#### COS Total Processing Fees
+
+Processing Cost
+Contains card components only
+
+#### Total Cost of Sales
+
+Total Cost
+:= Total Cost Card + Total Cost Noncard
+
+#### Gross Revenue
+
+Gross Revenue
+:= Gross Revenue Card + Gross Revenue Noncard
+Contains card and noncard components
+
+#### Gross Processing Revenue
+
+Gross Processing Revenue
+Contains card components only
+
+#### Net Revenue
+
+Net Revenue
+:= Net Revenue Card + Net Revenue Noncard
+Contains card and noncard components
+
+#### Net Processing Revenue
+
+Net Processing Revenue
+Contains card components only
+
+#### COS Association Fees, Assessments, and SWITCH Fees
+
+Association And Switch Fees Cost
+No card components
+
+#### COS Association Fees & Assessments
+
+Association Fees Cost
+
+#### COS SWITCH Fees
+
+Switch Fees Cost
+
+#### COS Interchange Fees
+
+Interchange Fees Cost
+No card components
+
+#### COS Other Processing Fees
+
+Other Fees Cost
+No card components
+
+#### Other COS
+
+Other Cost
+No card components
+
+#### Residuals Paid
+
+Residuals Cost
+No card components
+
+#### Legacy Account Annual Fees Revenue
+
+Legacy Account Annual Fees Revenue
+No card components
+
+#### Monthly Legacy Account Fees
+
+Legacy Account Monthly Fees Revenue
+No card components
+
+#### Discount Revenue
+
+Discount  Revenue
+Contains card components only
+
+#### Equipment & Other Income
+
+Equipment and Other Revenue
+Contains card components only
+
+#### Gross Profit
+
+Gross Profit Revenue
+Contains card components only
+
+#### Legacy Account Annual and Monthly Fees Revenue
+
+Legacy Account Annual and Monthly Fees Revenue
+Contains card components only
+
+#### Other Fee Revenue
+
+Other Fees Revenue
+No card components
+
+#### PCI Annual And Monthly Fees Revenue
+
+PCI Annual And Monthly Fees Revenue
+No card components
+
+#### Transaction Fee Revenue
+
+Transaction Fees Revenue
+No card components
+
+#### Transactions
+
+Transaction
+Contains card components only
+
+#### Volume
+
+Volume
+Contains card components only
+
+#### Account Attrition
+
+
+
+#### New Accounts Added
+
+
+
+#### Gross Volume Attrition
+
+
+
+#### Change in Retained Account Volume
+
+
+
+#### Volume Net Attrition
+
+
+
+#### New Volume Added
+
+
+
+#### Net Revenue Gross Attrition
+
+
+
+#### Change in Retained Account Net Revenue
+
+
+
+#### Net Revenue Net Attrition
+
+
+
+#### New Net Revenue Added
+
+
+
+#### Average Attrited Account Size
+
+
+
+#### Average Retained Account Size
+
+
+
+#### Average New Account Size
+
+
+
+#### Average Net Revenue BPS on Attrited Accounts
+
+
+
+#### Average Net Revenue BPS on Retained Accounts (Pre Change)
+
+
+
+#### Average Net Revenue BPS on Retained Accounts (Post Change)
+
+
+
+#### Average Net Revenue BPS on New Accounts
+
+</details>
+
+### Normalization
+
+Abstract Metric Normalizer.
```

Comment:
Al:
> What is normalization

Reply:
Normalization is the "Per X". The default is "Per Merchant". Most metrics also support "Per Transaction" and "Per Volume". The growth/attrition metrics have specific normalizations, and there are a few other special cases like Chargeback that don't support all normalizations.